pkg/kversion/kversion.go

```diff
@@ -337,7 +340,7 @@ func (vs *Versions) versionGuess(opts ...VersionGuessOpt) guess {
 		//
 		// TODO: add introduced-version to differentiate some specific
 		// keys.
-		skipKeys: []int16{4, 5, 6, 7, 27, 58},
+		skipKeys: []int16{4, 5, 6, 7, 27, 57, 64, 67},
```

Comment:
Why add UpdateFeatures? It's meant to be supported with/without Kraft. FWIW it's basically so far unused so I'm ok to remove it.

Envelope should still be filtered here, it's Kraft specific for the non-raft broker to forward a request to the raft broker when needed.

+1 for 64 and 67, also both Raft APIs, although this should also probably filter 59, 62, and 63

Reply:
`UpdateFeatures` is only used by brokers to update the list of finalized features or do I missunderstand this API? I think it's not used by clients and therefore I'd consider it part of the "internal" list, but maybe you had different criterias in mind what APIs should go into this list?

I actually thought 58 was a typo for 58, but I wasn't sure what the "Envelope" API does either.

Not sure what API `59`, `62` and `63` are, I can't find it in the Apache Kafka protocol docs and they were not reported by Apache Kafka 3.7.0 either.